### PR TITLE
Add scripts variant management script to FF & FE package json

### PR DESCRIPTION
**Pull request by kjose90**
#2326

- Added `start-variants-management` to FE and FF 
- Added a shared function `getVariantPreviewAppScript` to `fiori-generated-shared`
- Added Unit tests & updated snapshots

### CODE REVIEW

**Review by IainSAP — DISMISSED** [review]
Thanks @kjose90 

- Updates look correct to add script
- Tests added to cover

File: packages/fiori-generator-shared/src/helpers.ts [hunk 1]
```diff
@@ -29,3 +29,24 @@ export function getBootstrapResourceUrls(
...
+
+/**
+ * Generates a variant management script in preview mode.
+ *
+ * @param {string} sapClient - The SAP client parameter to include in the URL. If not provided, the URL will not include the `sap-client` parameter.
+ * @returns {string} A variant management script to run the application in preview mode.
+ */
+export function getVariantPreviewAppScript(sapClient?: string): string {
+    const previewAppAnchor = '#preview-app';
+    const DisableCacheParam = 'sap-ui-xx-viewCache=false';
```
Comment (longieirl) [inline packages/fiori-generator-shared/src/helpers.ts:41]: Should this be `disableCacheParam`?

**Review by longieirl — DISMISSED** [review]
Havent test locally.
Appropiate test changes to cover change.
Comments addressed.

**Review by IainSAP — APPROVED** [review]
Thanks @kjose90 

- Changes as expected
- Tests updated as expected

**Review by devinea — APPROVED** [review]
code changes are clear. Snapshot changes appear valid. 
Changeset ✅

**Review by longieirl — APPROVED** [review]
Re-approving.
A number of issues being followed up with another PR.

**Pull request merged**